lib/views/pages/profile/posts_info/popup/comment_popup.dart

```diff
@@ -0,0 +1,46 @@
+import "package:flutter/material.dart";
+
+class CommentPopUp extends StatelessWidget {
+  static const commentPopUpTitleKey = Key("commentPopUpTitle");
+  static const commentPopUpDescriptionKey = Key("commentPopUpDescription");
+  static const commentPopUpDeleteButtonKey = Key("commentPopUpDeleteButton");
+
+  const CommentPopUp({
+    super.key,
+    required this.comment,
+  });
+
+  final String comment;
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      title: const Text(
+        key: commentPopUpTitleKey,
+        "Comment",
```

Comment:
Maybe we shouldn't have a title "Comment" on the pop-up, since we don't have a title "Post" on the post pop-up, for consistency.

Reply:
You are right, removed in 6713997